ansible_collections/arista/avd/roles/eos_designs/python_modules/interface_descriptions/__init__.py

```diff
@@ -1,4 +1,33 @@
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get, load_python_class
+
 from .avdinterfacedescriptions import AvdInterfaceDescriptions
-from .load_interface_descriptions import load_interfacedescriptions
 
 __all__ = ["AvdInterfaceDescriptions", "load_interfacedescriptions"]
+
+DEFAULT_AVD_INTERFACE_DESCRIPTIONS_PYTHON_MODULE = "ansible_collections.arista.avd.roles.eos_designs.python_modules.interface_descriptions"
+DEFAULT_AVD_INTERFACE_DESCRIPTIONS_PYTHON_CLASS_NAME = "AvdInterfaceDescriptions"
+
+
+def load_interfacedescriptions(hostvars, templar) -> AvdInterfaceDescriptions:
```

Comment:
this should be in it's own file. Same for ip

Reply:
that makes a file to maintain for one function but will change this!